infrastructure/kubernetes/common/monitoring/metrics-scraper/deployment.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metrics-scraper
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: metrics-scraper
+  template:
+    metadata:
+      labels:
+        app: metrics-scraper
+    spec:
+      containers:
+        - name: metrics-scraper
+          image: metrics-scraper:latest
+          imagePullPolicy: IfNotPresent
```

We use `latest` so I think `Always` is better to always fetch the latest changes:
```suggestion
          imagePullPolicy: Always
```

---

addressed indeed: 0041b6d8c5e78e26f2651cd8d2ec085728b67a0e